distro/configs/openmrs/initializer_config/globalproperties/disable-demo-patients.xml

```diff
@@ -0,0 +1,8 @@
+<config>
+    <globalProperties>
+        <globalProperty>
+            <property>referencedemodata.createDemoPatientsOnNextStartup</property>
+            <value>0</value>
+        </globalProperty>
+    </globalProperties>
+</config>
```

Comment:
How is the community file enabling demo data creation overridden/deleted? When testing on gitpod.io, generation happened on startup instead of being triggered by the demo service.

Reply:
>When testing on gitpod.io, generation happened on startup instead of being triggered by the demo service.

Sorry ignore the above comment, it's all fine with the demo service.

Reply:
However where is [globalproperties_core-demo.xml](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/main/distro/configuration/globalproperties/globalproperties_core-demo.xml) being deleted from?

Reply:
Right here:
https://github.com/ozone-his/ozone-distro/blob/e9d2081d6a67b097c03c4c5ca386e4da44bb68c1/distro/pom.xml#L291-L310

I've configured to use Maven resources plugin to let us exclude files from O3 Ref App.
This is handy.